src/main/topLevelPages/myDashboard/newVirtualContributorWizard/newVirtualContributorQueries.graphql

```diff
@@ -25,6 +29,20 @@ query NewVirtualContributorMySpaces {
           id
         }
       }
+      community {
+        id
+      }
+    }
+  }
+}
+
+query NewSpace($spaceId: UUID_NAMEID!) {
```

Comment:
I'd call this `SpaceUrl`, and maybe there's already a similar query. 
Also if we have the ID (which I'm pretty sure we do), better access it through the lookup with UUID. if not please call the parameter $spaceNameId